spec/features/user_creates_review_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+feature 'user creates book review', %{
+As a user I want to create a review
+that is associated with a book
+so I can tell others what I think of the book
+} do
+# Acceptance Criteria:
+  #
+  # [√] A user should be able to title their review, required field.
+  # [√] A user should be able to leave a review no greater than 500 words
+  # [√] A user should be able to rate a book on a 1 to 5 scale
+  # [√] A user should be able to rate the book in their review on a 1 to 5 scale
+  # [√] A user should not be able to submit more than one review to a single book.
+  # [√] The review should be associated with a book.
+  # [√] The review should have a starting vote of 0.
+
+  let!(:author) { Author.create(name: "Jane Doe") }
+  let!(:genre) { Genre.create(genre_name: "Comedy") }
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:book) { Book.create(title: 'Book Title',
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.